projects/plugins/jetpack/extensions/blocks/videopress/video-chapters/components/details-control/index.js

```diff
@@ -30,7 +31,7 @@ export default function DetailsControl( { isRequestingVideoItem } ) {
 	};
 
 	return (
-		<PanelBody title={ __( 'Details', 'jetpack' ) }>
+		<PanelBody title={ __( 'Details', 'jetpack' ) } className="is-beta">
```

Comment:
it should be defined [via the Jetpack initial state](https://github.com/Automattic/jetpack/blob/c79e0ff8bf9cb4c4f3663a98ba2356e44739defb/projects/plugins/jetpack/extensions/blocks/videopress/video-chapters/index.js#L8).